README.md

```diff
@@ -1,43 +1,48 @@
+Documents in this repository are written in markdown. You can manually copy the information to your own help file.
 
-The repository also includes a [docfx.json](docfx.json) file, which enables you to convert a set of topics to an HTML website or a PDF file using [DocFX](https://dotnet.github.io/docfx/) and [wkhtmltopdf](https://github.com/wkhtmltopdf/wkhtmltopdf).   
+The repository also includes a [docfx.json](docfx.json) file. You can use [DocFX](https://dotnet.github.io/docfx/) and [wkhtmltopdf](https://github.com/wkhtmltopdf/wkhtmltopdf) to convert this file from a set of topics to an HTML website or a PDF file.   
 
 ## Build an HTML Website
-To create a documentation website for your application:
+Follow the steps below to create a documentation website for your application.
 
 - Download and install the [latest version of DocFX](https://github.com/dotnet/docfx/releases). 
-- Clone the repository to your computer and checkout the branch corresponding to the version of DevExpress controls your application uses. Make sure not to use the **master** branch, which represents the version currently under development.
+- Copy the repository to your computer and checkout the branch corresponding to the version of DevExpress controls your application uses. Do not use the **master** branch because it represents the version currently under development.
     ```
     git clone https://github.com/DevExpress/dotnet-eud.git --branch 18.1
     ```
-  If you do not have [Git](https://git-scm.com/) installed, select the required branch using the GitHub web interface and then download and extract the ZIP archive.
+  If you do not have [Git](https://git-scm.com/) installed, use the GitHub web interface to select the branch and then download and extract the ZIP archive.
   
   ![Download ZIP](https://user-images.githubusercontent.com/20167812/29712204-4ffaee9e-89a1-11e7-8a0e-3ff0464adda4.png)
-- If you want to reuse the end-user documentation as is, skip this step. Otherwise, make changes as required, which may include:
-  - removing files that are not needed in your documentation;
-  - adding new documents specific to your application;
-  - changing screenshots to match your app UI;
-  - creating a [custom DocFX template](https://dotnet.github.io/docfx/tutorial/howto_create_custom_template.html).
-  > Make sure to update *toc.yml* ([table-of-content](https://dotnet.github.io/docfx/tutorial/intro_toc.html)) files if you have added or removed topics.
-- Open a console window, change the current directory to the repository root folder and call the DocFX executable with the *build* and *docfx.json* parameters. DocFX will place the generated documentation content into the *\_site* folder. Add the *--serve* switch to preview the generated website at http://localhost:8080 once the build process is complete. 
+- You can skip this step if you want to reuse the end-user documentation as is. Otherwise, make changes as required, which may include:
```

Comment:
"make changes as required, which may include"?

Reply:
Yes. Below is a possible list of changes.

Reply:
You can skip this step if you want to reuse the end-user documentation as is. Otherwise, you make changes as required, which may include: ->
You can make the following changes to the documentation, or skip this step if you want to reuse the end-user documentation as is: